repos/spack_repo/builtin/packages/parallelio/package.py

```diff
@@ -65,11 +65,17 @@ class Parallelio(CMakePackage):
     depends_on("netcdf-fortran", type="link", when="+fortran")
     depends_on("parallel-netcdf", type="link", when="+pnetcdf")
 
-    resource(name="genf90", git="https://github.com/PARALLELIO/genf90.git", tag="genf90_200608")
+    resource(
+        name="genf90",
+        git="https://github.com/PARALLELIO/genf90.git",
+        tag="genf90_200608",
+        commit="4816965ba946731352bad195b7d946a5fe682ff5",
+    )
```

Comment:
@AlexanderRichert-NOAA can you describe this resource a little? It's from a very sparse GitHub org with only two stars on the repository. Is it something that absolutely cannot be upstreamed into parallelio?

Reply:
I don't know that it absolutely _can't_/couldn't be, I mean, it's a perl script, but as far as I can tell it's used by at least a handful of packages for Fortran 90 templating w.r.t. different data types etc. Since it's used by multiple packages I imagine the devs would not want to move into any particular repository. We could make it a build-type dependency but I don't know if that would make life any easier. I'm not a parallelio developer but I would lean toward leaving it as a resource.

Reply:
@AlexanderRichert-NOAA I cherry-picked this PR for https://github.com/jcsda/spack-packages